client/app/services/query.js

```diff
@@ -52,7 +52,7 @@ export class Query {
       this.options = {};
     }
     this.options.apply_auto_limit = !!this.options.apply_auto_limit;
-    this.options.apply_long_query = !!this.options.apply_long_query;
+    this.options.apply_long_query = false;
```

Comment:
@DebangshuB what does this change do?

Reply:
Currently, If we create a new query in Redash, the default state of the `Mark as a Long Query` is not consistent. This change defaults it to unchecked